cogs/fishing.py

```diff
@@ -0,0 +1,35 @@
+import discord
+import utils
+from discord.ext import commands
+
+
+class MembersCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.fish = bot.fish
+        self.db = utils.DatabaseConnection
```

Comment:
You define these and then don't use them.

Reply:
I was working on this but then got a headache and fell asleep